DEPS.bzl

```diff
@@ -2448,10 +2458,10 @@ def go_deps():
         name = "com_github_fraugster_parquet_go",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/fraugster/parquet-go",
-        sha256 = "66beb8f2218c31ca4ca3d3dbcc91c256f3379750ade924016f4179982446edd7",
-        strip_prefix = "github.com/fraugster/parquet-go@v0.6.1",
+        sha256 = "68aa69311763a3c816552da6b2571272f996bc111237cecbb171dd7d41bb9346",
+        strip_prefix = "github.com/fraugster/parquet-go@v0.8.0",
```

Comment:
Looks like the latest is 0.9.0, want to bump so close to their most recent release, but figured I'd point it out.

Reply:
yeah, there's a known regression in 0.9.0 they're working on :/

Hope to update to latest release during stability.